litmus-portal/backend/auth/go.sum

```diff
@@ -13,19 +13,25 @@ github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSs
 github.com/dgrijalva/jwt-go v3.2.0+incompatible h1:7qlOGliEKZXTDg6OTjfoBKDXWrumCAMpl/TFQ4/5kLM=
```

Comment:
Why those changes are here. Will you remove changes from go.sum

Reply:
These changes came in on starting the server @oumkale